frontend/test/pytest/test_measurements_shots_results.py

```diff
@@ -481,6 +481,109 @@ def circuit(theta):
         assert np.allclose(result, expected, atol=tol_stochastic, rtol=tol_stochastic)
 
 
+class TestVnEntropy:
```

Comment:
What is the difference between this test and the one in test_measurements_results.py? I assume the main difference is in setting `shots` in the device? 

If this is correct, are such measurements supported with finite shots on default.qubit?